bot.js

```diff
@@ -1,57 +1,138 @@
 const token = process.env.TOKEN
 const Bot = require('node-telegram-bot-api')
 const quiz = require('./quiz.js')
+const i18n = require('i18n')
+const defaultLocale = 'en'
+const supportedLocales = ['en', 'ru']
 
-let bot
+let bot = createBot()
 let storage = {}
+i18n.configure({
+  directory: 'data/locales/',
+  locales: supportedLocales,
+  defaultLocale: defaultLocale
+})
+
+function createBot () {
+  if (process.env.NODE_ENV !== 'production') {
+    return new Bot(token, { polling: true })
+  }
 
-if (process.env.NODE_ENV === 'production') {
-  bot = new Bot(token)
+  let bot = new Bot(token)
   bot.setWebHook(process.env.HEROKU_URL + bot.token)
-} else {
-  bot = new Bot(token, { polling: true })
+  return bot
 }
 
-function postNewQuestion (chatID) {
-  const question = quiz.generateQuestion('en')
-  storage[chatID] = { 'answer': question.answer }
+function localeForMsg (msg) {
+  const chatID = msg.chat.id
+  let locale = msg.from.language_code.toLowerCase()
+  if (supportedLocales.includes(locale) === false) {
+    locale = defaultLocale
+  }
+  return (storage[chatID] && storage[chatID].locale) || locale
+}
+
+function postNewQuestion (msg) {
```

Comment:
Let's avoid giving unnecessary info
```suggestion
function postNewQuestion (chatID, locale) {
```